docs/data/joy/components/table/table.md

```diff
@@ -0,0 +1,272 @@
+  </tr>
+</thead>
+```
+
+### Alignment
+
+Use `sx` prop to target columns with appropriate CSS selector and apply the [text-align](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align) property.
+
+```js
+// target cells that are not the first of its row.
+<Table sx={{ '& tr > *:not(:first-child)': { textAlign: 'right' } }}>
+```
+
+{{"demo": "TableAlignment.js"}}
+
+### Variants
+
+Table supports Joy UI's four [global variants](/joy-ui/main-features/global-variants/): `plain` (default), `outlined`, `soft`, and `solid`.
+
+{{"demo": "TableVariants.js"}}
+
+:::info
+To learn how to add your own variants, check out [Themed components—Extend variants](/joy-ui/customization/themed-components/#extend-variants).
+Note that you lose the global variants when you add custom variants.
+:::
+
+### Sizes
+
+The component comes in three sizes: `sm`, `md` (default), and `lg`.
+
+{{"demo": "TableSizes.js"}}
+
+:::info
+To learn how to add custom sizes to the component, check out [Themed components—Extend sizes](/joy-ui/customization/themed-components/#extend-sizes).
+:::
+
+### Stripe
+
+{{"demo": "TableStripe.js"}}
+
+:::success
+
+The prop also supports complex argument of [`:nth-child()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#syntax) CSS selector.
+
+For example, you can use `3n` as a value to create stripes on row number three, six, nine, and so on:
+
+```js
+<Table stripe="3n">
+```
+
+:::
+
+### Hover
+
+To highlight a row of the table body when hover on it, set `hoverAxis` prop to true.
+
+{{"demo": "TableHover.js"}}
+
+### Border
+
+Use `borderAxis` prop to control the border appearance.
+
+{{"demo": "TableBorder.js"}}
+
+<details>
+<summary><b>Adding custom values</b></summary>
+
+Customize the table theme based on `borderAxis` prop using [`extendTheme()`](/joy-ui/customization/themed-components/#change-styles-based-on-props) function.
+
+```js
+import { CssVarsProvider, extendTheme } from '@mui/joy/styles';
+
+const theme = extendTheme({
+  components: {
+    JoyTable: {
+      styleOverrides: {
+        root: ({ ownerState }) => ({
+          ...(ownerState.borderAxis === 'header' && {
+            '& thead th:not([colspan])': {
+              border: '2px solid var(--TableCell-borderColor)',
+            },
+          }),
+        });
+      }
+    }
+  }
+})
+
+<CssVarsProvider theme={theme}>…</CssVarsProvider>
+```
+
+For **TypeScript**, you have to add the new values via module augmentation:
+
+```ts
+// this could be any file that's included in your tsconfig.json
+declare module '@mui/joy/Table' {
+  interface TablePropsBorderAxisOverrides {
+    header: true;
+  }
+}
+```
+
+</details>
+
+### Sticky header
+
+Set `stickyHeader` prop to true. The table must be a child of a fixed-height element with overflow `auto` (or `scroll`).
+
+Using [Sheet](/joy-ui/react-sheet/) component is recommended for wrapping a table. See [usage with Sheet](#usage-with-sheet) to learn more.
```

Comment:
Maybe a `:::success` here? Since this is a recommendation for how to succeed with these components. :)

```suggestion
:::success
For `stickyHeader` to work correctly, the Table must be a child of a fixed-height element with overflow `auto` (or `scroll`).
We recommend wrapping your Table with [Sheet](/joy-ui/react-sheet/) for this purpose. 
See [usage with Sheet](#usage-with-sheet) to learn more.
:::
```